engines/advancedDetector.cpp

```diff
@@ -252,6 +261,30 @@ bool AdvancedMetaEngineDetection::cleanupPirated(ADDetectedGames &matched) const
 	return false;
 }
 
+bool AdvancedMetaEngineDetection::cleanupMissingResource(ADDetectedGames &matched) const {
+	// OKay, now let's sense presence of missing resource forks
+	if (!matched.empty()) {
+		for (uint j = 0; j < matched.size();) {
+			if (matched[j].hasMissingResourceFork)
+				matched.remove_at(j);
+			else
+				++j;
+		}
+
+		// We ruled out all variants and now have nothing
+		if (matched.empty()) {
+			warning("The dump is missing required resource forks. Please use dumper-companion.py or a similar tool");
+			if (GUI::GuiManager::hasInstance()) {
+				GUI::MessageDialog dialog(_("The dump is missing required resource forks. Please use dumper-companion.py or a similar tool"));
```

Comment:
Please drop "or a similar tool", and use `MessageDialogWithURL` pointing to `https://docs.scummvm.org/en/latest/use_scummvm/mac_game_files.html`